src/components/store/my-state.ts

```diff
@@ -14,6 +14,7 @@ export interface MyRecipesState {
 export interface MyRecipesInCartState {
   recipesInCart: string[];
   addRecipeToCart: (recipe: string) => void;
+  deleteRecipeFromCart: (recipe: string) => void;
```

Comment:
Co je vlastne to recipe: string? To je to Date.toString(36) ID? Pokud jo, tak pojdme pouzit neco jako 
```suggestion
  deleteRecipeFromCart: (recipe: NewRecipe['id']) => void;
```

At je to na prvni pohled jasny (a taky v pripadne, ze to zmenis, tak aby se ti to vsude propsalo zadarmo). Co ty na to?